packages/@coorpacademy-components/src/atom/provider/index.js

```diff
@@ -68,7 +68,7 @@ class Provider extends React.Component {
   static childContextTypes = {
     history: historyShape,
     skin: skinShape,
-    translate: translateShape.isRequired,
+    translate: translateShape,
```

Comment:
I removed it because it is not forcefully required anymore as in mobile's case, it is not compulsory present on native components that used shared basic components